lib/cocoapods/generator/embed_frameworks_script.rb

```diff
@@ -76,10 +76,12 @@ def script
             local swift_runtime_libs
             swift_runtime_libs=$(xcrun otool -LX "${CONFIGURATION_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/$1/${basename}" | grep @rpath/libswift | sed -E s/@rpath\\\\/\\(.+dylib\\).*/\\\\1/g | uniq -u  && exit ${PIPESTATUS[0]})
             for lib in $swift_runtime_libs; do
-              echo "rsync -av \\"${SWIFT_STDLIB_PATH}/${lib}\\" \\"${destination}\\""
-              rsync -av "${SWIFT_STDLIB_PATH}/${lib}" "${destination}"
-              if [ "${CODE_SIGNING_REQUIRED}" == "YES" ]; then
-                code_sign "${destination}/${lib}"
+              if [ -n "${destination}" ]; then
```

Comment:
:laughing:

Even if it was, that check would cause only the **first** Swift library to be copied and code signed. Since `destination` is a constant directory in which files are copied to. Once the directory exists, no more libraries would be copied into it.

I think you'd want something like this instead:

```bash
if [ -r "${destination}/${lib}" ]; then
```